demisto_sdk/commands/validate/tests/GR_validators_test.py

```diff
@@ -529,3 +532,147 @@ def test_IsUsingUnknownContentValidator__different_dependency_type__list_files(
+    Then:
+        - The validator should pass, everything is valid, sanity check
+    """
+    graph_interface = repo_test_from_version.create_graph()
+    BaseValidator.graph_interface = graph_interface
+    results = (
+        IsUsingInvalidFromVersionValidator().obtain_invalid_content_items_using_graph(
+            content_items=[]
+        )
+    )
+    assert len(results) == 0
+
+
+def test_IsUsingInvalidFromVersionValidator_invalid(
+    repo_test_from_version: Repo,
+):
+    """
+    Given:
+        - A content graph interface with preloaded repository data:
+            - Pack 1:
+                - playbook (which uses a command from the integration)
+                - integration (which contains a command, used by the playbook)
+            - Pack 2:
+                    - script 1 (which used by script 2, but has fromversion=10.0.0 while script 2 has fromversion=0.0.0)
+                    - script 2 (which uses script 1)
+    When:
+        - The GR101 validation is executed across the second script
+    Then:
+        - The validator should fail due to target's fromversion higher than source's fromversion. (len(results) == 1)
+        - Ensure the error message as expected
+    """
+    repo_test_from_version.packs[1].scripts[0].set_data(
+        **{"fromversion": "10.0.0"}
+    )  # This line fails the GR101
+    graph_interface = repo_test_from_version.create_graph()
+    BaseValidator.graph_interface = graph_interface
+    results = (
+        IsUsingInvalidFromVersionValidator().obtain_invalid_content_items_using_graph(
+            content_items=[repo_test_from_version.packs[1].scripts[1]]
+        )
+    )
+    assert len(results) == 1
+    assert (
+        results[0].message
+        == "Content item 'UsingDeprecatedScript' whose from_version is '0.0.0'"
+        " is using content items: 'DeprecatedScript' whose from_version is higher"
+        " (should be <= 0.0.0)"
+    )
+
+
+def test_IsUsingInvalidFromVersionValidator_valid(
+    repo_test_from_version: Repo,
+):
+    """
+    Given:
+        - A content graph interface with preloaded repository data:
+            - Pack 1:
+                - playbook (which uses a command from the integration)
+                - integration (which contains a command, used by the playbook)
+            - Pack 2:
+                    - script 1 (which used by script 2, has fromversion=10.0.0)
+                    - script 2 (which uses script 1, has fromversion=11.0.0)
+    When:
+        - The GR101 validation is executed across the second script
+    Then:
+        - The validator should pass, since script 2 which uses script 1 has a higher fromversion, valid case
+    """
+    repo_test_from_version.packs[1].scripts[0].set_data(**{"fromversion": "10.0.0"})
+    repo_test_from_version.packs[1].scripts[1].set_data(**{"fromversion": "11.0.0"})
+    graph_interface = repo_test_from_version.create_graph()
+    BaseValidator.graph_interface = graph_interface
+    results = (
```

Comment:
very nice!.
Pease change your test, to call the "suns" of the validation instead the father.
some examples with all files and some with list of files.
so this test should call:

```suggestion
        results = IsUsingInvalidFromVersionValidatorListFiles.obtain_invalid_content_items(content_items=[repo_test_from_version.packs[1].scripts[1]])
```